scripts/bluepill.sh

```diff
@@ -82,12 +81,28 @@ bluepill_build()
           echo No bp built
           exit 1
   }
+  xcodebuild \
```

Comment:
Should you need to build this explicitly? I thought it was built as a dependency of bluepill?

Reply:
Xcode was having difficulty building it with the right architecture when I add it as a target dependency. Let me give it another shot and if it doesn't work out, building it separately is not bad either.